src/main/java/edu/ithaca/gamemaster/Player.java

```diff
@@ -73,42 +73,258 @@
+                character.getWisdom(),character.getCharisma(),character.getAlignment(),character.getLanguageList(),
+                character.getActionsList());
+
+        //can be set by this constructor or be set elsewhere based on class and race
+        this.acrobatics=acro;
+        this.animalHandling=animalHandling;
+        this.arcana=arcana;
+        this.athletics=athletics;
+        this.deception=deception;
+        this.history=history;
+        this.insight=insight;
+        this.intimidation=intimidation;
+        this.investigation=investigation;
+        this.medicine=medicine;
+        this.nature=nature;
+        this.perception=perception;
+        this.performance=performance;
+        this.persuasion=persuasion;
+        this.religion=religion;
+        this.slightOfHand=slightOfHand;
+        this.stealth=stealth;
+        this.survival=survival;
+
+        this.passiveWisdom=passiveWisdom;
+        this.initiative=initiative;
+        this.age=age;
+        this.size=size;
+        this.charClass=charClass;
+        this.level=level;
+        this.race=race;
+
+        this.hasAbility=true;
+
     }
 
     public Player(Character character){
-        super(character.getName());
+        super(character.getName(),character.getHP(),character.getHitDiceAmt(),
+                character.getHitDiceSided(),character.getHitDiceModifier(),character.getArmor(),
+                character.getSpeed(),character.getStrength(),character.getDexterity(),
+                character.getConstitution(),character.getIntelligence(),
+                character.getWisdom(),character.getCharisma(),character.getAlignment(),character.getLanguageList(),
+                character.getActionsList());
+    }
+
+    public Player(String name,int hitPts, int hitDice, int hitSide, int hitMod,
+                  int armor, int speed, int strength, int dexterity,
+                  int constitution, int intelligence, int wisdom, int charisma,
+                  String alignment, ArrayList<String> languages, ArrayList<Action> actions){
+        super(name,hitPts,hitDice,hitSide,hitMod,armor,speed,strength,dexterity,constitution
+        ,intelligence,wisdom,charisma,alignment,languages,actions);
+    }
+
+    //Generators
+
+    public void generateSkills(){
+        this.acrobatics=dexteSave;
+        this.animalHandling=wisdoSave;
+        this.arcana=intelSave;
+        this.athletics=strenSave;
+        this.deception=chariSave;
+        this.history=intelSave;
+        this.insight=wisdoSave;
+        this.intimidation=chariSave;
+        this.investigation=intelSave;
+        this.medicine=wisdoSave;
+        this.nature=intelSave;
+        this.perception=wisdoSave;
+        this.performance=chariSave;
+        this.persuasion=chariSave;
+        this.religion=intelSave;
+        this.slightOfHand=dexteSave;
+        this.stealth=dexteSave;
+        this.survival=wisdoSave;
+
+        for(int i=0;i<skills.size();i++){
```

Comment:
There is probably a more time efficient way to do this but this works perfectly well, yo.